core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -396,6 +396,11 @@ public static ByteString concat(ByteString s0, ByteString s1) {
     return s0.concat(s1);
   }
 
+  /** SQL {@code string compare string} operator. */
+  public static Integer strcmp(String s0, String s1) {
+    return s0.compareTo(s1);
+  }
```

Comment:
I thin It is better to return `int` instead of `Integer`.

Reply:
The doc is confusing, what is `{@code string compare string} operator` ?

Reply:
had changed it.